lib/cli/src/frameworks/react/js/Page.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Header } from './Header';
+import './page.css';
+
+export const Page = ({ user, onLogin, onLogout, onCreateAccount }) => (
+  <article>
+    <Header user={user} onLogin={onLogin} onLogout={onLogout} onCreateAccount={onCreateAccount} />
+
+    <section>
+      <h2>Pages in Storybook</h2>
+      <p>
+        We recommend building UIs with a{' '}
+        <a
+          href="https://blog.hichroma.com/component-driven-development-ce1109d56c8e"
```

Comment:
```suggestion
          href="http://componentdriven.org/"
```

@tmeasday can you set up componentdriven.org to redirect for now?

Reply:
Already did @domyen

Reply:
Is the redirect working for you? http://componentdriven.org

Reply:
That's weird it was on Friday

Reply:
I tried all sorts of variations, only link that works is http://www.componentdriven.org/

Reply:
I think I fixed it.